v2/features/stats/platform/common.tsx

```diff
@@ -0,0 +1,67 @@
+import { PickleModelJson } from "picklefinance-core";
+import { ReceiptDataModel, VerboseReceipts } from "./types";
+import { RawChain } from "picklefinance-core/lib/chain/Chains";
+
+const PICKLE_API = `https://f8wgg18t1h.execute-api.us-west-1.amazonaws.com/prod`;
+
+const formatMoney = (bigMoney: number): string => {
+  return "$" + bigMoney.toFixed(2).replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+};
+
+const formatPct = (pct: number): string => {
+  return pct.toFixed(3) + "%";
+};
+
+const formatMoneyChange = (bigMoney: number): string => {
+  return bigMoney > 0 ? 
+    '+' + bigMoney.toFixed(2).replace(/\B(?=(\d{3})+(?!\d))/g, ",") :
+    bigMoney.toFixed(2).replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+};
+
+const formatPctChange = (change: number) => {
+  return change > 0 ?
+    "+ " + (change * 100).toFixed(2) + " %" :
+    (change * 100).toFixed(2) + " %";
+};
+
+const getCurrentReceipts = async (): Promise<ReceiptDataModel> => {
+  return await fetch(`${PICKLE_API}/protocol/receipts/current`).then((response) => response.json());
+};
+
+const getLastReceipts = async (): Promise<ReceiptDataModel> => {
+  return await fetch(`${PICKLE_API}/protocol/receipts/last`).then((response) => response.json());
+};
+
+const getCurrentReceiptsVerbose = async (): Promise<VerboseReceipts> => {
+  return await fetch(`${PICKLE_API}/protocol/receipts/current/verbose`).then((response) => response.json());
+};
+
+const getLastReceiptsVerbose = async (): Promise<VerboseReceipts> => {
+  return await fetch(`${PICKLE_API}/protocol/receipts/last/verbose`).then((response) => response.json());
+};
+
+const getCurrentPfCore = async (): Promise<PickleModelJson.PickleModelJson> => {
+  return await fetch(`${PICKLE_API}/protocol/pfcore/current`).then((response) => response.json());
+};
+
+const getLastPfCore = async (): Promise<PickleModelJson.PickleModelJson> => {
+  return await fetch(`${PICKLE_API}/protocol/pfcore/last`).then((response) => response.json());
+};
+
+export {
+  formatMoney,
+  formatPct,
+  formatMoneyChange,
+  formatPctChange,
+  getCurrentReceipts,
+  getLastReceipts,
+  getCurrentReceiptsVerbose,
+  getLastReceiptsVerbose,
+  getCurrentPfCore,
+  getLastPfCore,
+};
```

Comment:
I suggest you don't mix this pattern with individual exports. I'd favor exports in front of any exported entity.

Reply:
deleted file